device/nokia/x86_64-nokia_ixr7250_x3b-r0/platform_env.conf

```diff
@@ -3,3 +3,4 @@ dmasize=512M
 default_mtu=9100
 macsec_enabled=1
 SYNCD_SHM_SIZE=512m
+disaggregatedT2=1
```

Comment:
can we change this to disaggregated chassis here as well?

Reply:
Yes. Change is done